tests/test_dimensions.py

```diff
@@ -6,10 +6,10 @@
 def test_dimensions():
     dims = ansunits.BaseDimensions
     d1 = ansunits.Dimensions(dimensions_container={})
-    assert d1.dimensions == {}
+    assert d1._dimensions == {}
 
     d2 = ansunits.Dimensions(dimensions_container={dims.MASS: 1.0, dims.LENGTH: -2.0})
```

Comment:
I'm guessing that we had called this `dimensions_container` to support both `list` and `dict`. We can simply now, which helps the code to document itself.